app/metrics/admin.py

```diff
@@ -10,14 +10,17 @@
     Answer,
     Response,
     ResponseSet,
-    UserProfile
+    UserProfile,
+    Node
 )
 
 
 EDIT_TRACKING_FIELDS = [
```

Comment:
I think this should be renamed to `COMMON_EXCLUDES` in order to communicate better what the intention is.